main.sh

```diff
@@ -9,15 +9,15 @@ echo_with_date "当前 Oh My WeChat 版本为 v${omw_version}"
 
 # 从 GitHub 获取 owm 版本号
 get_omw_latest_version_from_github() {
-  curl --retry 2 -I -s https://github.com/lmk123/oh-my-wechat/releases/latest | grep -i Location: | sed -n 's/.*\/v\(.*\)/\1/p'
+  curl --retry 2 -I -s https://github.com/lmk123/oh-my-wechat/releases/latest | grep -i Location: | sed -n 's/.*\/[v,V]\(.*\)/\1/p'
 }
 
 get_download_url() {
   echo https://github.com/MustangYM/WeChatExtension-ForMac/archive/v${1}.zip
```

Comment:
The download link is still in lowercase v. Does this cause a problem?

I would actually suggest that we should get the version name instead of comparing numbers.